db-sdc/controllers.js

```diff
@@ -0,0 +1,71 @@
+const { Client } = require('pg');
```

Comment:
It is better that you use pool instead of Client. It is one the optimizations that can be done. Look at this: https://node-postgres.com/features/pooling